lib/web_ui/dev/test_runner.dart

```diff
@@ -204,7 +207,7 @@ class TestCommand extends Command<bool> with ArgUtils {
 
     // If screenshot tests are available, fetch the screenshot goldens.
     if (isScreenshotTestsAvailable) {
-      print('screenshot tests available');
+      print('INFO: Screenshot tests available');
```

Comment:
I'd remove this message. It just adds noise during the normal usage of the `felt`. If this is useful for debugging the felt tool itself, I'd add a `--verbose` option for such messages.

Reply:
Thanks makes sense.

Reply:
done.